### ISSUE TITLE
Change max loansizes to be smaller, interest to be higher and max loan count to be smaller. This is to stop people from farming loans to mass produce buildings. 

### ISSUE THREAD

**Issue by Dragonkn1ght** [RohanTLK/Make-Europa-Great-Again#25]
This is dumb

**Comment by ComradeL1berty** [comment]
Interest could be a bit higher imo. 3% for most people barely matters.

**Comment by SpectreOfRohan** [comment]
Yeah maybe just interest being higher is fine. It's far too easy to go into loan debt in this game and come out of it economically ahead. Especially manufactury spam

**Comment by Dragonkn1ght** [comment]
IMO that is sort of the point...Maybe move the base to 5-6 or something if you really want, but not crazy high.

**Comment by SpectreOfRohan** [comment]
It's 6 at the moment. 
I think it's detrimental to game strategy if you have a huge safety net of loans to fall back on. I think it's a better show of strategy if you manage your economy well enough to stay out of deep debt.
